DisguiseUnityRenderStream/Runtime/PipelineAbstraction/BlitExtended.cs

```diff
@@ -0,0 +1,73 @@
+using UnityEngine;
+using UnityEngine.Assertions;
+using UnityEngine.Rendering;
+
+class BlitExtended
+{
+    public enum ColorSpaceConversion
+    {
+        None,
+        LinearToSRGB,
+        SRGBToLinear
+    }
+    
+    static class ShaderIDs
+    {
+        public static readonly int _BlitTexture = Shader.PropertyToID("_BlitTexture");
+        public static readonly int _BlitScaleBias = Shader.PropertyToID("_BlitScaleBias");
+    }
+    
+    static BlitExtended s_Instance;
+
+    public static BlitExtended Instance
+    {
+        get
+        {
+            if (s_Instance == null)
+                s_Instance = new BlitExtended();
+
+            return s_Instance;
+        }
+    }
+
+    public static Vector4 IdentityScaleBias { get; } = new Vector4(1f, 1f, 0f, 0f);
+    public static Vector4 FlippedYScaleBias { get; } = FlipYScaleBias(IdentityScaleBias);
+    public static Vector4 FlipYScaleBias(Vector4 scaleBias) => new Vector4(scaleBias.x, -scaleBias.y, scaleBias.z, 1f - scaleBias.w);
+
```

Comment:
Minor: It took me some time (and looking in the shaders) to understand how those bias are working.  Maybe add a small comment somewhere to say that xy is the scale and zw the offset?

Reply:
Added a `ScaleBias` struct in `BlitStrategy.cs` to document the behavior and abstract away the vector packing